R/transformAbundance.R

```diff
@@ -0,0 +1,400 @@
+#' Transform Abundance
+#'
+#' This function applies transformation to abundance table, which is returned.
+#' Additionally, the information can be directly stored in the \code{assay}.
+#'
+#' @param x
+#' A \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
+#' object.
+#'
+#' @param abund_values
+#' A single character value for selecting the
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{assay}}
+#'   to be transformed.
+#'
+#' @param transform
+#' A single character value for selecting the transformation method.
+#'
+#' @param name
+#' A single character value specifying the name of transformed abundance table.
+#'
+#' @param pseudocount
+#' Boolean or numeric value deciding whether pseudocount is added. Numerical
+#' value specifies the value of pseudocount. When selecting pseudocount==TRUE,
+#' pseudocount is 1.
```

Comment:
I think this should accept, FALSE and and any numeric value. `TRUE` and then translating it two `1` is probably to specific. It robes the user of an option, which is easy to implement

Reply:
Pseudocount can now get FALSE and any numeric values.